src/tests/rest-calls-200-404.spec.ts

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+import { AxiosHarTracker } from '../axios-har-tracker'
+import { writeFileSync } from 'fs';
+
+describe('Check axios-har-tracker for status 200 and 404', () => {
+
+    let axiosTracker, getHar
+
+    beforeAll(async () => {
+      axiosTracker = new AxiosHarTracker(axios);
+    });
+
+    afterAll(() => {
+      writeFileSync('example200404.har', JSON.stringify(getHar), 'utf-8')
+    });
+
+    it('should get har which includes status 200', async () => {
+      await axios.get('http://httpstat.us/200');
+      getHar = axiosTracker.getGeneratedHar();
+      const array = getHar.log.entries;
+      expect(array[0].request).toMatchObject({
+        "method": "get",
+        "url": "http://httpstat.us/200"
+      });
+      expect(array[0].response).toMatchObject({
+        "status": 200,
+        "statusText": "OK"
+      });
+      expect(array[1]).toBeUndefined();
+    });
+
+    it('should get har which includes status 404 and all previous requests with responses', async () => {
+      try {
+        await axios.get('http://httpstat.us/404');
+      } catch(error){
+        console.log("An error appears:",error);
+      }
+      getHar = axiosTracker.getGeneratedHar();
+      const array = getHar.log.entries;
+      expect(array[0].request).toMatchObject({
+        "method": "get",
+        "url": "http://httpstat.us/200"
+      });
+      expect(array[0].response).toMatchObject({
```

Comment:
Don't combine calls from two different `it` tests. If you want to check if multiple calls are stored in one generatedHar objects, do both calls in the same test case. So for example you could have 3 test cases:

- it('should collect call with status 200') -> here you make only call for 200
- it('should collect call with status 404') -> here you make only call for 404
- it('should collect multiple calls") -> here you make two calls for 200 and 404 and check if array contains both entries